test/shaders/glsl/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+#######################
+cmake_minimum_required(VERSION 3.15)
+
+vulkan_compile_shader(INFILE test_logistic_regression_shader.comp
+                      OUTFILE test_logistic_regression_shader.hpp
+                      NAMESPACE "kp")
+
+vulkan_compile_shader(INFILE test_op_custom_shader.comp
+                      OUTFILE test_op_custom_shader.hpp
+                      NAMESPACE "kp")
+
+vulkan_compile_shader(INFILE test_workgroup_shader.comp
+                      OUTFILE test_workgroup_shader.hpp
+                      NAMESPACE "kp")
+
+vulkan_compile_shader(INFILE test_shader.comp
+                      OUTFILE test_shader.hpp
+                      NAMESPACE "kp")
+
+add_library(test_shaders_glsl INTERFACE "${CMAKE_CURRENT_BINARY_DIR}/test_logistic_regression_shader.hpp"
```

Comment:
Interesting - it could make sense in this one to add as separate library, but does feel like it could be easier if this was just a glob for the existing project that didn't require adding explicitly ever hpp file and every comp shader, which was one of the reasons to have the `shaders/` directory for the tests, perhaps could be one that can be simplified?

Reply:
Yes, was thinking about something like this as well, but I decided against it since in my eyes explicit is always better than implicit. This way we enforce the user directly not to be lazy from the beginning. This is in my eyes one of the big points that speak against something like this, besides the fact that glob is being considered bad style in CMake.

Reply:
I do see what you mean, I wonder if there could be two interfaces for this (one that can use a glob), but I think that can be explored as a followup